jdi-light-bootstrap-tests/src/test/java/io/github/epam/bootstrap/tests/composite/section/form/FormRowTests.java

```diff
@@ -0,0 +1,58 @@
+package io.github.epam.bootstrap.tests.composite.section.form;
+
+import io.github.epam.TestsInit;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+import static io.github.com.StaticSite.bsPage;
+import static io.github.com.pages.BootstrapPage.formRow;
+import static io.github.epam.bootstrap.tests.BaseValidations.baseValidation;
+import static io.github.epam.states.States.shouldBeLoggedIn;
+import static org.hamcrest.Matchers.is;
+
+public class FormRowTests extends TestsInit {
+
+    private String text_first_name = "First name";
+    private String text_last_name = "Last name";
+    private String text_for_send_keys = "test send keys";
+
+    @BeforeMethod
+    public void before() {
+        shouldBeLoggedIn();
+        bsPage.shouldBeOpened();
+    }
+
+
+    @Test
+    public void checkStructureRow() {
+        formRow.row.assertThat().hasClass("form-row");
+        formRow.row.is().enabled().displayed();
+        formRow.row.childs().foreach(e -> {
+            e.hasClass("col");
```

Comment:
it's not a check, you need is() here, also you can you chaining to concatenate it with displayed and enabled:
```
formRow.row.childs().foreach(e -> {
            e.is()
                  .hasClass("col")
                  .displayed()
                  .enabled();
        });
```

Reply:
fixed.

as childs() return not only closest children, but also children of children - check .hasClass("col") removed